src/transaction.ts

```diff
@@ -0,0 +1,234 @@
+import { Psbt, networks } from 'liquidjs-lib';
+import { estimateTxSize, UtxoInterface } from './wallet';
+
+export interface OutputInterface {
+  value: number;
+  asset: string;
+  script: string;
```

Comment:
Would use the `address` instead of script, how a normal end-user would manage it